docs/core/preview3/tools/dotnet-new.md

```diff
@@ -15,45 +15,87 @@ ms.assetid: fcc3ed2e-9265-4d50-b59e-dc2e5c190b34
 #dotnet-new (Tooling RC3)
 
 > [!WARNING]
-> This topic applies to Visual Studio 2017 RC - .NET Core Tools RC3. For the .NET Core Tools Preview 2 version,
+> This topic applies to Visual Studio 2017 RC - .NET Core Tools RC4. For the .NET Core Tools Preview 2 version,
 > see the [dotnet-new](../../tools/dotnet-new.md) topic.
 
 ## Name
-dotnet-new -- Creates a new .NET Core project in the current directory.
+dotnet-new - Creates a new .NET Core project in the current directory
 
 ## Synopsis
-`dotnet new [--help] [--type] [--lang]`
+`dotnet new [arguments] [options]`
```

Comment:
we should the different synopses for each template as we discussed for dotnet sln, no?